README.md

```diff
@@ -53,16 +244,141 @@ Optionally, use the command line tool to generate example records for Canary tes
 
 You can also output XML instead of JSON by passing the ```--xml``` option.
 
+
+# Creating death record
+
+VRDRFhirContext ctx = new VRDRFhirContext();
```

Comment:
Please put all these sections of code inside a code block as is done in the vrdr-dotnet library README